<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>636</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtTiffViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>871</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTiffData">
     <attribute name="title">
      <string>Tiff данные</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaTiffData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>421</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsTiffData">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>419</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidgetTiffData">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>849</width>
          <height>419</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabGeoData">
     <attribute name="title">
      <string>Geo данные</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetGeoData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>851</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Key directory version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="keyDirectoryVersionLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Keys version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="keyVersionLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of keys:</string>
      </property>
     </widget>
     <widget class="QLabel" name="numbKeysLabel">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper left:</string>
      </property>
     </widget>
     <widget class="QLabel" name="uLLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>741</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower left:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lLLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>721</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper right:</string>
      </property>
     </widget>
     <widget class="QLabel" name="uRLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>721</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower right:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lRLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>731</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Center:</string>
      </property>
     </widget>
     <widget class="QLabel" name="centerLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>731</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonViewContents">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Посмотреть содержимое</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>586</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Страница </string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevPageBt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPageBt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BigTiff:</string>
    </property>
   </widget>
   <widget class="QLabel" name="isBigTiffLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего тегов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numTagsLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
